akka-cluster-sharding/src/main/mima-filters/2.5.x.backwards.excludes

```diff
@@ -3,3 +3,13 @@ ProblemFilters.exclude[Problem]("akka.cluster.sharding.Shard.*")
 
 # #25191
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.cluster.sharding.ShardRegion.retryTask")
+
+# #27100 Productionize: GetShardRegionStats returns empty shard set on ask timeout
+# askAllShards, an internal function, was renamed and changed to query all or a subset of shards to try failures only
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.cluster.sharding.ShardRegion.askAllShards")
+# Added new field for failed shard queries to ShardRegion#ShardRegionStats, converted to class and updated in proto
+ProblemFilters.exclude[MissingTypesProblem]("akka.cluster.sharding.ShardRegion$ShardRegionStats$")
```

Comment:
I've not seen errors related to that being true, and there was no companion object before. But I am not yet a MIMA expert, it was reported as needed.

Reply:
> it was reported as needed

Let's say it was reported as broken and needs justification to be ignored. :)

Reply:
Missing types {scala.runtime.AbstractFunction1}

Reply:
Note in the excludes file added.

Reply:
The companion object is generated automatically for case classes. It seems that generated companion object implements `Function1`, i.e. there is a binary incompatibility, the question would be if people would run into it (probably low).

A simple way to fix it would be to add `extends AbstractFunction1[Map[ShardId, Int], ShardRegionStats]` to the new `ShardRegionStats` companion object and remove the line here.

Reply:
Works! Pushed.

Reply:
Thanks @jrudolph, good to know about this, and all of your review suggestions were very helpful.

Reply:
> Works! Pushed.

Great :)

Reply:
> Thanks @jrudolph, good to know about this, and all of your review suggestions were very helpful.

Glad it helped, started to feel bad, bugging you with all of this. :)